cypress/integration/login.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Login Page', () => {
+  beforeEach(() => {
+    cy.visit('/login');
+  });
+
+  it('displays login form', () => {
+    cy.get('form').should('be.visible');
+  });
+
```

Comment:
Consider using a helper function to fill out the form. This will reduce repeated code and make it easier to make changes to the form in the future.